configure.ac

```diff
@@ -9,8 +9,8 @@ dnl Custom macro definitions.
 
 dnl JE_CONCAT_VVV(r, a, b)
 dnl
-dnl Set $r to the concatenation of $a and $b, with a space separating them iff
```

Comment:
`iff` here is actually used in a technical sense; it's an abbreviation for "if and only if". The "emty" fix looks good, though.